package.json

```diff
@@ -639,9 +639,11 @@
 		"nyc": "^15.1.0",
 		"run-sequence": "^2.2.1",
 		"source-map-resolve": "0.6.0",
+		"ts-mock-imports": "^1.3.3",
```

Comment:
I think we need to update package-lock.json too

Reply:
Why? The package-lock.json does not need to upload, right?

Reply:
Please check the history of package-lock.json, we've been updating this file all the time.